scipy/optimize/_hessian_update_strategy.py

```diff
@@ -0,0 +1,338 @@
+"""quasi-Newton update strategies."""
```

Comment:
Really sorry to bring this up again, but as you wrote it "quasi" should be upper case as the first word.

Reply:
Ok!